mediawiki.json

```diff
@@ -1,21 +1,45 @@
 {
-  "extends": [
-    "./vue-es5.js"
-  ],
   "globals": {
     "mw": "readonly",
     "require": "readonly",
     "module": "readonly"
   },
   "plugins": [ "mediawiki" ],
   "rules": {
-    "no-implicit-globals": "off",
     "mediawiki/class-doc": "error",
     "mediawiki/msg-doc": "error",
-    "mediawiki/no-vue-dynamic-i18n": "error",
-    "mediawiki/valid-package-file-require": "error",
-    "vue/v-bind-style": [ "error", "longform" ],
-    "vue/v-slot-style": [ "error", "longform" ],
-    "vue/v-on-style": [ "error", "longform" ]
-  }
+    "mediawiki/valid-package-file-require": "error"
+  },
+  "overrides": [
+    {
+      "files": [ "**/*.vue" ],
+      "extends": [
+        "plugin:vue/recommended",
+        "./vue-es5.js"
+      ],
+      "plugins": [ "es" ],
+      "parserOptions": {
```

Comment:
You shouldn't need to touch parserOptions here, vue-es5.js already does this

Reply:
Ah, yeah, was just hacking locally until it worked.